src/main/java/org/apache/maven/plugins/dependency/analyze/AbstractAnalyzeMojo.java

```diff
@@ -266,7 +267,30 @@ public abstract class AbstractAnalyzeMojo
     // defaultValue value on @Parameter - not work with Maven 3.2.5
     // When is set defaultValue always win, and there is no possibility to override by plugin configuration.
     @Parameter
-    private List<String> ignoredPackagings = Arrays.asList( "pom", "ear" );
+    private final List<String> ignoredPackagings = Arrays.asList( "pom", "ear" );
+
+
+    /**
+     * List of dependencies to be included. The result of this list is then applied to the ignore parameters.
```

Comment:
This isn't clear that everything is included by default when this isn't set.

Reply:
Yes, I've been musing on this point. The default behaviour prior to this change is that everything is included. That can't change without requiring a pom changes downstream. So this is strictly:
 
"_an override to the ignore_"

That's a bit mind-bending if you don't have the context. I think the best we can do is to provide a recipe in the Javadoc describing an example use case, such as the one we're going to use it for. Broadly, everything _ignored_, but this one Maven co-ordinate _in_.

It doesn't feel like if we were designing this anew we'd start from here, but without a brain-wave on how to prevent a backward incompatible change, this will be the approach.

Reply:
Thinking through some of the use-cases further and wanting more fine-grained control, we're essentially building sub-graphs in the dependency graph. I think that is expressible using Maven co-ordinates, but would be good to back these thoughts with some tests.

Reply:
Decided that building sub-graphs was way to complicated, so sticking with the original changes, which is consistent with the standard Maven filters.... [include then exclude](https://github.com/apache/maven-common-artifact-filters/blob/9d692d49b35014a2d308c78894ee502bccd0361e/src/main/java/org/apache/maven/shared/artifact/filter/collection/AbstractArtifactFeatureFilter.java#L63).